09-navigate-safari-voiceover.md

```diff
@@ -0,0 +1,17 @@
+ # 09. Navigate a Webpage with VoiceOver in Safari
+
+**[📹 Video](https://egghead.io/lessons/screen-reader-navigate-a-webpage-with-voiceover-in-safari)**
+
+
+* VoiceOver works best with MacOS' native browser - Safari.
+
+* The keyboard shortcut ⌨️ for enabling VoiceOver is `cmd + f5`.
+
+The VoiceOver commands will consist of the VO or VoiceOver Activation key(typically `control + option` keys ⌨️ ), followed by another key or two.
+
+| keys ⌨️   | command            |
```

Comment:
Love it!

Reply:
Thanks for showing me the 'Add suggestion to batch' - I've never done it before 👍